src/um/secext.rs

```diff
@@ -0,0 +1,59 @@
+// Licensed under the Apache License, Version 2.0
+// <LICENSE-APACHE or http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your option.
+// All files in the project carrying such notice may not be copied, modified, or distributed
+// except according to those terms.
+use shared::minwindef::PULONG;
+use um::winnt::{BOOLEAN, LPSTR, LPWSTR};
+ENUM!(
```

Comment:
This should use the following style:
```suggestion
ENUM!{enum EXTENDED_NAME_FORMAT {
    NameUnknown = 0,
}}
```